### PR TITLE
Fix new global nav issues

### PR DESCRIPTION
**Pull request by mirabelrice**

### CODE REVIEW

File: .cascade-code/Chapman.edu/_config/gloabal_nav.vtl [hunk 1]
```diff
@@ -0,0 +1,222 @@
+#set ($GLOBAL_NAV_CONFIG = [
```
Comment (tatwell) [inline .cascade-code/Chapman.edu/_config/gloabal_nav.vtl]: Should this be name `global_nav.vm`?

File: .cascade-code/Chapman.edu/_config/gloabal_nav.vtl [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        'link': '/about/administration/index',
+        'iconClass': 'icon-cu-window'
+      },
+      {
+        'text': 'Contact Us',
+        'link': '/about/contact-us',
+        'iconClass': 'icon-envelop'
+      }
+    ]
+  },
```
Comment (tatwell) [inline .cascade-code/Chapman.edu/_config/gloabal_nav.vtl]: I like this structure. 👍

File: .cascade-code/Chapman.edu/_config/global_nav_law.vtl [hunk 3]
```diff
@@ -0,0 +1,214 @@
+#set ( $GLOBAL_NAV_LAW_CONFIG = [
```
Comment (tatwell) [inline .cascade-code/Chapman.edu/_config/global_nav_law.vtl]: Should this have `.vm` extension? I'm comparing to [this line](https://github.com/chapmanu/cascade-assets/pull/281/files#diff-9eb228b5664fc3bdcb787f84d31f02bbR17).

Comment (mirabelrice) [reply .cascade-code/Chapman.edu/_config/global_nav_law.vtl]: yes. fixed.

File: .cascade-code/Chapman.edu/_config/gloabal_nav.vm [hunk 4]
```diff
@@ -0,0 +1,222 @@
+#set ($GLOBAL_NAV_CONFIG = [
```
Comment (tatwell) [inline .cascade-code/Chapman.edu/_config/gloabal_nav.vm]: You still have a typo in the file name: `gloabal`

**Pull request merged**